app/screens/FollowInfoStorage.js

```diff
@@ -2,7 +2,7 @@ import {AsyncStorage} from "react-native"
 import { STRINGS } from '../assets/constants.js'
 
 export async function getToken() {
-    return await AsyncStorage.getItem('token') || "" ;
+    return await AsyncStorage.getItem('token') || (__DEV__ ? "tsd-dev": "") ;
```

Comment:
I’m not sure if this would help development? The token is assigned by Expo server, not our local variable.

Reply:
I think it helps because at least then the `updateBackend()` function works (otherwise it gets a 404 when calling the REST API)